examples/model_assisted_labeling/mal_basics.ipynb

```diff
@@ -2,73 +2,84 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install labelbox
</pre><p>Not needed since there is <code>pip install labelbox[data]</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>